.editorconfig

```diff
@@ -0,0 +1,24 @@
+# This file is for unifying the coding style for different editors and IDEs
```

Comment:
What is this? I don't think we should commit editor settings along with the repo?